week2/1-tuesday/class/exercise8/exercise8.js

```diff
@@ -13,4 +13,24 @@ function numberCounter(numbers){
     return console.log("Positive Numbers: " + positiveCount + '\n' + "Negative Numbers: " + negativeCount);
 }
 
-numberCounter(randomNumbers);
\ No newline at end of file
+numberCounter(randomNumbers);
+
+// @caitlincraw : pretty much did the same thing besides created an empty object at the beginning and then added the summations to it. 
+
+function sums(nums) {
+    var summationOfSigns = {};
+    var sumOfNeg = 0;
+    var sumOfPos = 0;
+    for (var i = 0; i < nums.length; i++) {
+        if (nums[i] < 0) {
+            sumOfNeg = sumOfNeg + nums[i];
+        } else if (numsList[i] > 0) {
+            sumOfPos = sumOfPos + nums[i];
+        }
+    }
```

Comment:
Consider `forEach` and `+=` here if you want to make it more concise. Also I think you might have a typo with `numsList`